pkg/webhook/v1beta1/experiment/validator/validator_test.go

```diff
@@ -454,6 +455,30 @@ func TestValidateParameters(t *testing.T) {
 			},
 			testDescription: "Not empty max for categorical parameter type",
 		},
+		{
+			parameters: func() []experimentsv1beta1.ParameterSpec {
+				ps := newFakeInstance().Spec.Parameters
+				ps[0].FeasibleSpace.Distribution = "invalid-distribution"
+				return ps
+			}(),
+			wantErr: field.ErrorList{
+				field.Invalid(field.NewPath("spec").Child("parameters").Index(0).Child("feasibleSpace").Child("distribution"), "", ""),
+			},
+			testDescription: "Invalid distribution type",
+		},
+		{
+			parameters: func() []experimentsv1beta1.ParameterSpec {
+				ps := newFakeInstance().Spec.Parameters
+				ps[0].FeasibleSpace = experimentsv1beta1.FeasibleSpace{
+					Max:          "5",
+					Min:          "1",
+					Distribution: experimentsv1beta1.DistributionUniform,
+				}
+				return ps
+			}(),
```

Comment:
```suggestion
			parameters: newFakeInstance() ,
```

We already set the valid parameters in the `newFakeInstance` function, right?